src/otk/command.py

```diff
@@ -8,6 +8,7 @@
 from .context import registry as context_registry
 from .document import Omnifest
 from .help.log import JSONSequenceHandler
+from .parserstate import ParserState
```

Comment:
I don't want to do naming discussions but I also can't not say this: it isn't the state of the parser, `traversal.State` or such would make more sense to me. Feel absolutely free to ignore.

Reply:
Thanks, that is a good point, ParserState may not be ideal, `{traversal,recursion,rec,trans}.State` are some alternative ideas (fwiw, I don't have a super strong opinion here)

Reply:
I renamed the module to `traversal` and the class to `State`.